staging/src/k8s.io/apiserver/pkg/apis/config/types.go

```diff
@@ -24,7 +24,23 @@ import (
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// EncryptionConfiguration stores the complete configuration for encryption providers.
+/*
+EncryptionConfiguration stores the complete configuration for encryption providers.
+example:
+
+	kind: EncryptionConfiguration
+	apiVersion: apiserver.config.k8s.io/v1
+	resources:
+	  - resources:
+		- secrets
+		- configmaps
+		- pandas.awesome.bears.example
+		providers:
+		- aescbc:
+			keys:
+			- name: key1
+			  secret: c2VjcmV0IGlzIHNlY3VyZQ==
```

Comment:
I don't think this indentation renders correctly... this currently looks like this:

![image](https://user-images.githubusercontent.com/980082/207059027-81c2eb83-c1b8-4613-9dce-9905fb987c51.png)

Swapping tabs for spaces and aligning `- resources` and `providers:`
```suggestion
    kind: EncryptionConfiguration
    apiVersion: apiserver.config.k8s.io/v1
    resources:
    - resources:
      - secrets
      - configmaps
      - pandas.awesome.bears.example
      providers:
      - aescbc:
          keys:
          - name: key1
            secret: c2VjcmV0IGlzIHNlY3VyZQ==
```
produces this:

![image](https://user-images.githubusercontent.com/980082/207059278-0ff5ece3-169d-4029-a89c-4d7eb5304920.png)

Reply:
(same comment for the v1 types file)

Reply:
Thanks for the suggestion! PTAL